src/components/AppNavigationSpacer/AppNavigationSpacer.vue

```diff
@@ -20,7 +20,7 @@
  -
  -->
 <template>
-	<li />
+	<li class="spacer" />
```

Comment:
Let's keep the same class structure? :see_no_evil: 
`app-navigation-spacer` ?

Reply:
sure!